core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -396,8 +408,11 @@ public double getMaxspeed(EdgeIteratorState edge, int accessType, boolean revers
 
     @Override
     public long flagsDefault(boolean forward, boolean backward) {
-        // just pick car mode to set access values?
-        throw new RuntimeException("do not call flagsDefault");
+//         just pick car mode to set access values?
+//        throw new RuntimeException("do not call flagsDefault");
+    	// TODO This is called on each of the encoders so I had to replace the runtime exception with something,
+    	// but I'm not sure this is correct.
+    	return setAccess(0, forward, backward);
```

Comment:
Ah, yes. One would need the carFwdFlagEncoder here ... but we also have backward, hmmh. I would like to get rid of this.